components/statsigsdk/StatsigTask.brs

```diff
@@ -122,6 +122,9 @@ function StatsigNetwork(sdkKey, user) as object
                 user: m._user.toEvaluationDictionary()
             })) then
                 msg = Wait(3000, m._port)
+                if msg = invalid then
```

Comment:
FWIW, this is what we ended up doing on our end to address this for now

-                values = parseJSON(msg)
-                if values["feature_gates"] <> invalid and values["dynamic_configs"] <> invalid then
-                    return values
-                else
-                    return {feature_gates: {}, dynamic_configs: {}}
+
+                if type(msg) = "roUrlEvent" then
+                    data = msg.GetString()
+
+                    values = parseJSON(data)
+                    if GetInterface(values, "ifAssociativeArray") <> invalid then
+                        if values["feature_gates"] <> invalid and values["dynamic_configs"] <> invalid then
+                            return values
+                        end if
+                    end if
                 end if
+
+                return {feature_gates: {}, dynamic_configs: {}}